src/data/sources.json

```diff
@@ -7,58 +7,16 @@
                     "title": "OECD, 2016"
                 }
             },
-            "allPopulationSex2006S": {
-              "source": {
-                "href": "",
-                "title": ""
-              }
-            },
-            "allPopulationSex2007S": {
-              "source": {
-                "href": "",
-                "title": ""
-              }
-            },
             "allPopulationSex2009S": {
               "source": {
-                "href": "",
-                "title": ""
-              }
-            },
-            "allPopulationSex2011S": {
-              "source": {
-                "href": "",
-                "title": ""
-              }
-            },
-            "allPopulationSex2012S": {
-              "source": {
-                "href": "",
-                "title": ""
-              }
-            },
-            "allPopulationSex2013S": {
-              "source": {
-                "href": "",
-                "title": ""
+                "href": "https://s3-eu-west-1.amazonaws.com/s3.sourceafrica.net/documents/21195/Census-2009.pdf",
```

Comment:
We should link to sourceAFRICA page that _contains_ the PDF rather than liking to the PDF directly.